### PR TITLE
updated contributing.html

### PR DESCRIPTION
**Pull request by omkreddy**
updated "contributing website changes" section.

### CODE REVIEW

File: contributing.html [hunk 1]
```diff
@@ -25,9 +25,8 @@
 
 To submit a change for inclusion please do the following:
 
-<ul>
-	<li>Create a patch that applies cleanly against <a href="http://svn.apache.org/repos/asf/kafka/site/">SVN trunk</a>.</li>
-	<li>Open a <a href="https://issues.apache.org/jira/browse/KAFKA">JIRA</a> ticket describing the patch and attach your patch to the JIRA. Include in the JIRA information about the issue and the approach you took in fixing it (if this isn't obvious). Also, mark the jira as "Patch Available" by clicking on the "Submit Patch" button (this is done automatically if you use the code review tool below). Creating the JIRA will automatically send an email to the developer email list alerting us of the new issue.</li>
+<ul>    
+         <li>Follow the detailed instructions in <a href="https://cwiki.apache.org/confluence/display/KAFKA/Contributing+Website+Documentation+Changes">Contributing Website Changes</a>.</li>
 	<li>It is our job to follow up on patches in a timely fashion. <a href="mailto: dev@kafka.apache.org">Nag us</a> if we aren't doing our job (sometimes we drop things). If the patch needs improvement, the reviwer will mark the jira back to "In Progress" after reviewing.</li>
```
Comment (ijuma) [inline contributing.html]: Can you please fix the existing typo in this line (`reviwer`)?

File: contributing.html [hunk 2]
```diff
@@ -25,9 +25,8 @@
 
 To submit a change for inclusion please do the following:
 
-<ul>
-	<li>Create a patch that applies cleanly against <a href="http://svn.apache.org/repos/asf/kafka/site/">SVN trunk</a>.</li>
-	<li>Open a <a href="https://issues.apache.org/jira/browse/KAFKA">JIRA</a> ticket describing the patch and attach your patch to the JIRA. Include in the JIRA information about the issue and the approach you took in fixing it (if this isn't obvious). Also, mark the jira as "Patch Available" by clicking on the "Submit Patch" button (this is done automatically if you use the code review tool below). Creating the JIRA will automatically send an email to the developer email list alerting us of the new issue.</li>
+<ul>    
+         <li>Follow the detailed instructions in <a href="https://cwiki.apache.org/confluence/display/KAFKA/Contributing+Website+Documentation+Changes">Contributing Website Changes</a>.</li>
```
Comment (ijuma) [inline contributing.html]: Seems like this line is over-indented. Also, I'd probably drop `detailed` and just say "Follow the instructions..."

**Comment by ijuma** [comment]
In the page, we also have:

```
    <li>Note that if the change is related to user-facing protocols / interface / configs, etc, you need to make the corresponding change on the documentation as well. For wiki page changes feel free to edit the page content directly (you may need to contact us to get the permission first if it is your first time to edit on wiki); for website doc changes please follow the steps below to submit another patch as well, except it can be under the same JIRA as the code change and you do not need to create a new JIRA for it.
```

I think we should update that to say that it should be done as part of the same PR since the the docs now live under the code repo.

**Comment by guozhangwang** [comment]
LGTM, thanks @omkreddy .

**Pull request closed without merging**